.github/workflows/release-package-connector-template.yml

```diff
@@ -0,0 +1,91 @@
+name: Publish Release (Connector)
+
+on:
+  workflow_call:
+    inputs:
+      package-name:
+        required: true
+        type: string
+      package-org:
+        required: true
+        type: string
+      additional-build-flags:
+        required: false
+        type: string
+        default: ""
+      additional-publish-flags:
+        required: false
+        type: string
+        default: ""
+
+jobs:
+  publish-release:
+    name: Release Package
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'ballerina-platform'
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 17.0.7
+
+      - name: Build without Tests
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
+          git config --global user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
+          ./gradlew build -x check -x test ${{ inputs.additional-build-flags }}
+
+      - name: Create lib Directory if not Exists
+        run: mkdir -p ballerina/lib
+
+      - name: Run Trivy Vulnerability Scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: "rootfs"
+          scan-ref: "/github/workspace/ballerina/lib"
```

Comment:
I don't think this is applicable to the connectors.

Reply:
removed